fly_away/mapModule.ts

```diff
@@ -15,14 +15,12 @@ export const map = L.map("map", {
 
 //OpenSkyAPI Subscriber
 flightPositions$.subscribe((flights) => {
-  console.log(flights);
   markers = [];
   if (buttList) {
```

Comment:
Ambiguous.

Reply:
Is this the latest branch? Cause I can't find that console.log in the typescript branch.

Reply:
buttList: is it a list of buttons or a list of butts.

Personally, I prrefer the latter. But ideally, go for the first to avoid ambiguity

Reply:
I promise I'll mature for the React project.